api/v1beta1/disruption_webhook.go

```diff
@@ -107,9 +107,10 @@ func (r *Disruption) ValidateUpdate(old runtime.Object) error {
 		return fmt.Errorf("error getting new disruption hash: %w", err)
 	}
 
-	logger.Infow("comparing disruption spec hashes", "instance", r.Name, "namespace", r.Namespace, "oldHash", oldHash, "newHash", newHash)
+	logger.Debugw("comparing disruption spec hashes", "instance", r.Name, "namespace", r.Namespace, "oldHash", oldHash, "newHash", newHash)
 
 	if oldHash != newHash {
+		logger.Errorw("error when comparing disruption spec hashes", "instance", r.Name, "namespace", r.Namespace, "oldHash", oldHash, "newHash", newHash)
```

Comment:
Maybe worth using a warning here. This error can be triggered by a human by editing the disruption so it could easily flood logs and trigger a monitor of too many errors on our side.

Reply:
If it's triggered by a human, why would it flood logs? If it's a manual kubectl apply or kubectl edit, they should get rejected immediately